pkgs/by-name/ge/get-graphql-schema/package.nix

```diff
@@ -15,23 +19,18 @@
     hash = "sha256-ujc0LGAqmo4SmItm4VcbBOtmUvL6aV1ppMm4fMmuSRs=";
   };
 
-  packageJSON = ./package.json;
-  offlineCache = fetchYarnDeps {
+  yarnOfflineCache = fetchYarnDeps {
     yarnLock = "${src}/yarn.lock";
     hash = "sha256-TZGNX8UHbolLyBmQNGTnFjgx3/3f2HNVQf/h9rIVJKs=";
   };
 
-  buildPhase = ''
-    runHook preBuild
-    yarn --offline build
-    runHook postBuild
-  '';
+  nativeBuildInputs = [
+    yarnConfigHook
+    yarnBuildHook
+    npmHooks.npmInstallHook
+    nodejs
+  ];
 
-  postFixup = ''
-    substituteInPlace $out/libexec/get-graphql-schema/deps/get-graphql-schema/dist/index.js \
-      --replace-fail "#!/usr/bin/env node" "#!${lib.getExe nodejs_22}"
-    chmod +x $out/bin/get-graphql-schema
-  '';
   meta = {
     description = "Command line tool for generating a changelog from git tags and commit history";
     homepage = "https://github.com/cookpete/auto-changelog";
```

Comment:
Drive-by comment: These seem to be the wrong meta attribute values for this package?

Reply:
Yes they are, I must have mis-copied them while doing this for the auto-changelog package. Will fix this, thanks for noticing.

Reply:
> Yes they are, I must have mis-copied them while doing this for the auto-changelog package. Will fix this, thanks for noticing.

Should be fixed in a separate commit :).

Reply:
Will do!

Reply:
Fixed